docs/guides/firmware-update.md

```diff
@@ -44,6 +44,13 @@ After the firmware update is finished, the device will reboot automatically. If
 the update includes an Embedded Controller firmware update, it will be applied
 automatically after reboot and the device will reboot again.
 
+!!! warning
+
+    Keep in mind that the firmware will be updated automatically to the latest
+    Dasharo release and that you will not be able to provide DPP credentials.
```

Comment:
If one cannot provide credentials then FUM is close to useless... What is the blocker to let the DPP credentials be provided @artur-rs ?

Reply:
Issue for the discussion: https://github.com/Dasharo/dasharo-issues/issues/1211